NativeApp/src/components/Login/Form/index.js

```diff
@@ -16,67 +23,49 @@ class LoginForm extends Component {
     this.state = {
       email: '',
       password: '',
-      underlineColor1: 'gray',
-      underlineColor2: 'gray',
     };
   }
 
-  changeColor = (id, color) => {
-    if (id === 'input-1') {
-      this.setState({
-        underlineColor1: color,
-      });
-    } else {
-      this.setState({
-        underlineColor2: color,
-      });
-    }
-  }
-
   render() {
-    const { email, password, underlineColor1, underlineColor2 } = this.state;
+    const { email, password } = this.state;
     const { handleLogin, hasError, loading } = this.props;
-    const { changeColor } = this;
     return (
-      <View>
-        <TextInput
-          id="input-1"
-          onFocus={() => changeColor('input-1', '#00DCFA')}
-          onBlur={() => changeColor('input-1', 'gray')}
-          placeholder="Email"
-          onChangeText={text => this.setState({ email: text })}
-          value={email}
-          autoCapitalize="none"
-          underlineColorAndroid="transparent"
-          selectionColor="#00DCFA"
-          style={{ borderBottomWidth: 1, borderColor: hasError ? 'red' : underlineColor1 }}
-        />
-        <TextInput
-          id="input-2"
-          onFocus={() => changeColor('input-2', '#00DCFA')}
-          onBlur={() => changeColor('input-2', 'gray')}
-          placeholder="Password"
-          onChangeText={text => this.setState({ password: text })}
-          value={password}
-          secureTextEntry
-          underlineColorAndroid="transparent"
-          style={{ borderBottomWidth: 1, borderColor: hasError ? 'red' : underlineColor2 }}
-          selectionColor="#00DCFA"
-        />
-        {
-          hasError && (
-            <Caption type="base" style={styles.red}>
-              Incorrect email or password
-            </Caption>
-          )
-        }
+      <View style={styles.view}>
+        <View>
+          <FormLabel>Email</FormLabel>
+          <Input
+            onChangeText={text => this.setState({ email: text })}
+            placeholder="you@email.com"
+            value={email}
+            returnKeyType="next"
+            hasError={hasError}
+            onSubmitEditing={() => this.secondInput.focus()}
+            blurOnSubmit={false}
+          />
+          <FormLabel>Password</FormLabel>
+          <Input
+            textInputRef={(ref) => { this.secondInput = ref; }}
+            onChangeText={text => this.setState({ password: text })}
+            value={password}
+            returnKeyType="done"
+            placeholder="*******"
```

Comment:
Hard to know for sure without seeing the app itself, but I think this placeholder could be confusing? Better left empty IMO.

Reply:
![login_page](https://user-images.githubusercontent.com/40429630/43967878-78bcc8f4-9cbd-11e8-8d60-bf136e0cbebd.png)
![login_placeholder](https://user-images.githubusercontent.com/40429630/43967879-79061e64-9cbd-11e8-8250-ed44b496bb78.png)

I have made the changes as per your suggestion :)